src/BlockTypes/ProductButton.php

```diff
@@ -15,57 +15,6 @@ class ProductButton extends AbstractBlock {
 	 */
 	protected $block_name = 'product-button';
 
-	/**
-	 * API version name.
-	 *
-	 * @var string
-	 */
-	protected $api_version = '2';
-
-	/**
-	 * Get block supports. Shared with the frontend.
-	 * IMPORTANT: If you change anything here, make sure to update the JS file too.
-	 *
-	 * @return array
-	 */
-	protected function get_block_type_supports() {
```

Comment:
Removed the API version and the `get_block_type_supports()` as we are now registering this block via `block.json` (metadata).